packages/react-native/ReactCommon/react/renderer/components/view/AccessibilityPrimitives.h

```diff
@@ -34,6 +34,21 @@ enum class AccessibilityTraits : uint32_t {
   Header = (1 << 15),
   Switch = (1 << 16),
   TabBar = (1 << 17),
+// [macOS
```

Comment:
If this is macOS-only, should it be if-def'd?

Reply:
Perhaps this is also a candidate to upstream, since these cpp primitives should ideally be cross platform I think